docs/Project.toml

```diff
@@ -1,4 +1,10 @@
 [deps]
+DifferentialEquations = "0c46a032-eb83-5123-abaf-570d42b7fbaa"
```

Comment:
Why not keep it an optional dependency?

Reply:
only for the docs. i assume we'll use it

Reply:
Ah, I see :+1: